src/main/java/org/myrobotlab/framework/Service.java

```diff
@@ -786,11 +786,11 @@ public String getDataInstanceDir() {
   }
 
   public String getResourceDir() {
-    return "data" + File.separator + getClass().getSimpleName();
+    return "resource" + File.separator + getClass().getSimpleName();
   }
 
   public String getResourceInstanceDir() {
-    return "data" + File.separator + getClass().getSimpleName() + File.separator + getName();
+    return "resource" + File.separator + getClass().getSimpleName() + File.separator + getName();

```

Comment:
same comment about using the Util.getResourceDir() method instead of it being hard coded here.

Reply:
fixed ... added Util.getDataDir() for consistency too